src/Cards/CartCard.js

```diff
@@ -0,0 +1,86 @@
+import "./ProductCard.css";
+import { useMyReducer } from "../stateContext";
+import { BsTrashFill } from "react-icons/bs";
+import { BsPlus } from "react-icons/bs";
+import { FiMinus } from "react-icons/fi";
+import { Link } from "react-router-dom";
+import { useLanguageContext } from "../AllContext/languageContext";
+import { GetPercentageOff } from "../Logic";
+export const CartCard = (item) => {
+  const {
+    id,
+    imageSrc,
+    productName,
+    description,
+    price,
+    originalPrice,
+    newFlag,
+    quantity
+  } = item;
+  const { dispatch } = useMyReducer();
+  const { language } = useLanguageContext();
+  const moveToWishListFromCart = (id) => {
+    dispatch({
+      type: "update-wishlist",
+      payload: { item: item, id: id }
+    });
+    dispatch({ type: "remove-from-cart", payload: { id: id } });
+  };
+  return (
+    <div className="card">
+      <div className="product-details">
+        <Link to={`/products/${id}`}>
+          <img className="product-image" alt="product" src={imageSrc} />
+          {newFlag && <div className="float-new">{language.new}</div>}
+        </Link>
+        <div className="product-name-and-desc">
+          <Link to={`/products/${id}`}>
+            <h3 className="product-name-style">{productName}</h3>
+
+            <div className="price-details">
+              <strong>Rs. {price}</strong>
+              <strike className="price-strike-style">
+                Rs. {originalPrice}
+              </strike>
+              <span className="red-text">
+                {GetPercentageOff(originalPrice, price)}% {language.off}
```

Comment:
As this is a function try to use camelCase over here
```suggestion
                {getPercentageOff(originalPrice, price)}% {language.off}
```